config/environments/production.rb

```diff
@@ -34,6 +34,10 @@
   # creates false positive results.
   config.action_dispatch.ip_spoofing_check = false
 
+  if FeatureManagement.log_to_stdout?
+    Rails.logger = Logger.new(STDOUT)
+    config.logger = ActiveSupport::Logger.new(STDOUT)
+  end
```

Comment:
I just realized, this may cause a load error (because `FeatureManagement` may not have been loaded here)

Since at this point in time we don't need to do the Rails env check (since this file is literally only run in the produciton env), let's just make this a plain Figaro dependency:

```
if Figaro.env.log_to_stdout == 'true'
  Rails.logger = Logger.new(STDOUT)
  config.logger = ActiveSupport::Logger.new(STDOUT)
end
```

Reply:
Is that also true for the ahoy config?

Reply:
No, `config/initializers/*` are run after this IIRC, and those can use the Rails autoloader I think